scripts/sign_image.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -ex
+
+
+IMAGE=""
+SIGNING_KEY=""
+SIGNING_CERT=""
+CA_CERT=""
+TARGET_PATH=""
+TMP_CERT_PATH="/tmp/cert"
+CERT_PATH=""
+
+usage()
+{
+    echo "Usage:  $0 -i <image_path> [-c <cert_path>] [-t <cert_target_path>]"
+    exit 1
+}
+
+generate_signing_key()
+{
+    SIGNING_CSR="${TMP_CERT_PATH}/signing.csr"
+    CA_KEY="${TMP_CERT_PATH}/ca.key"
+    mkdir -p "${TMP_CERT_PATH}"
```

Comment:
Since they are temporary and internal, can we use `mktemp`?

Reply:
Fixed